Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -152,6 +155,23 @@ public static class GoogleAuthConsts
         /// </summary>
         internal static string EffectiveMetadataServerUrl => GetEffectiveMetadataUrl(null, DefaultMetadataServerUrl);
 
+        /// <summary>
+        /// Retry strategy recommended for the OAuth2 token endpoints.
+        /// </summary>
+        internal static readonly ExponentialBackOffInitializer OAuth2TokenEndpointRecommendedRetry = new ExponentialBackOffInitializer(
```

Comment:
Suggest making this a property instead of an internal field. (Just by adding `{ get; }`)

Reply:
Oh, yes